lib/orbital/models/response.rb

```diff
@@ -79,6 +79,18 @@ def self.orbital_response_params(response)
           :params_tx_ref_num => extract(response, 'tx_ref_num'),
         }
       end
+
+      def first_reference_id
+        params_tx_ref_num
+      end
+
+      def second_reference_id
+        params_order_id
+      end
+
+      def gateway_error_code
+        params_resp_code
```

Comment:
Do you know if that matches https://github.com/killbill/killbill-cybersource-plugin/blob/master/lib/cybersource/models/response.rb#L124 (when Orbital is used through CyberSource)?